src/main/java/mate/controller/DeleteCarController.java

```diff
@@ -0,0 +1,27 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+
+public class DeleteCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private CarService carService;
+
+    @Override
+    public void init() throws ServletException {
+        carService = (CarService) injector.getInstance(CarService.class);
+    }
+
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long id = Long.valueOf(req.getParameter("id"));
+        carService.delete(id);
+        resp.sendRedirect(req.getContextPath() + "/index");
+    }
```

Comment:
in previous comments you should decide which redirect would be better
but here redirect to `/cars` after deletion seems to be logical